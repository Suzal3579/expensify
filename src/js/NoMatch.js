import React from 'react';

const NoMatch = () => (
    <div >
        <h1>Unicorns exists? NO </h1>
        <h1>Neither does this page.</h1>
    </div>
);

export default NoMatch;
