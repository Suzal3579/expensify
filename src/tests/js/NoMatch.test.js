import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import NoMatch from '../../js/NoMatch';

test('should render NoMatch component', () => {
    const wrapper = shallow(<NoMatch />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});
