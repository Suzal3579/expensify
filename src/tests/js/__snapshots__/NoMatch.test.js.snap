// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoMatch component 1`] = `
<div>
  <h1>
    Unicorns exists? NO 
  </h1>
  <h1>
    Neither does this page.
  </h1>
</div>
`;
